app/controllers/stages_controller.rb

```diff
@@ -132,9 +132,11 @@ def stage_params
       :datadog_tags,
       :update_github_pull_requests,
       :comment_on_zendesk_tickets,
+      :email_committers_on_automated_deploy_failure,
+      :static_emails_on_automated_deploy_failure,
       command_ids: [],
       flowdock_flows_attributes: [:id, :name, :token, :_destroy],
-      new_relic_applications_attributes: [:id, :name, :_destroy]
+      new_relic_applications_attributes: [:id, :name, :_destroy],
```

Comment:
with an extra comma it's only a 1 line change when adding a new line vs
previous line changed (+,) and new line added

On Tue, Mar 3, 2015 at 4:07 PM, Roman Sandler <notifications@github.com>
wrote:

> In app/controllers/stages_controller.rb
> <https://github.com/zendesk/samson/pull/258#discussion_r25741211>:
>
> >        command_ids: [],
> >        flowdock_flows_attributes: [:id, :name, :token, :_destroy],
> > -      new_relic_applications_attributes: [:id, :name, :_destroy]
> > +      new_relic_applications_attributes: [:id, :name, :_destroy],
>
> can drop the comma
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/258/files#r25741211>.
>